sopel/builtins/adminchannel.py

```diff
@@ -366,24 +365,35 @@ def topic(bot, trigger):
 @plugin.require_privilege(plugin.OP, ERROR_MESSAGE_NO_PRIV)
 @plugin.command('tmask')
 def set_mask(bot, trigger):
-    """Set the topic mask to use for the current channel
+    """Set or get the topic mask to use for the current channel.
 
-    Within the topic mask, {} is used to allow substituting in chunks of text.
+    This mask is used when running the 'topic' command. `{}` is used to allow
+    interpolating chunks of text within the topic mask.
 
-    This mask is used when running the 'topic' command.
+    If called without arguments, prints the channel's current topic mask.
     """
-    bot.db.set_channel_value(trigger.sender, 'topic_mask', trigger.group(2))
+    new_mask = trigger.group(2)
+
+    if new_mask is None:
+        mask = bot.db.get_channel_value(
+            trigger.sender, 'topic_mask', default_mask(trigger))
+        bot.reply('Current topic mask: {}'.format(mask))
+        return
+
+    bot.db.set_channel_value(trigger.sender, 'topic_mask', new_mask)
     message = (
         'Topic mask set. '
-        'Use `{prefix}topic <args>` to set topic '
-        'and `{prefix}showmask` to see current mask.'
+        'Use `{prefix}topic <args>` to set topic, '
+        '`{prefix}tmask` without arguments to see the current mask, '
+        'and `{prefix}cleartmask` to clear the topic mask.'
     ).format(prefix=bot.settings.core.help_prefix)
     bot.reply(message)
 
 
 @plugin.require_chanmsg
 @plugin.require_privilege(plugin.OP, ERROR_MESSAGE_NO_PRIV)
-@plugin.command('showmask')
-def show_mask(bot, trigger):
-    """Show the topic mask for the current channel."""
-    bot.say(bot.db.get_channel_value(trigger.sender, 'topic_mask', default_mask(trigger)))
+@plugin.command('cleartmask')
```

Comment:
What do you say about that alias?

```suggestion
@plugin.command('cleartmask', 'ctmask')
```

Reply:
My general preference is still one command - No arguments to show the current mask, empty quotes or `-clear` to clear, and content to set; or get(or empty)/set/clear subcommands.
Otherwise, agreed - I don't think clearing the mask is something that will need to be done often, so I don't see a significant benefit to a shorter alias. I would end up looking it up every time anyway.